src/language/dafnyLanguageClient.ts

```diff
@@ -51,6 +52,14 @@ function getMarkGhostStatementsArgument(): string {
   return `--ghost:markStatements=${Configuration.get<string>(ConfigurationConstants.LanguageServer.MarkGhostStatements)}`;
 }
 
+function getDafnyPluginsArgument(): string {
+  let plugins = Configuration.get<string[]>(ConfigurationConstants.LanguageServer.DafnyPlugins);
+  if(plugins === null) {
+    return '';
+  }
+  return `--dafny:plugins=${plugins.join(',')}`;
```

Comment:
But on the Dafny side, don't we want to pass multiple copies of the flag instead of using a `,`?

Reply:
Indeed, I forgot to push my changes for this VSCode plugin